ote/src/cljs/ote/views/transit_visualization.cljs

```diff
@@ -71,18 +72,22 @@
      (when (and prev-week-hash hash (not= hash prev-week-hash) (> day (time/now)))
        {:box-shadow "inset 0 0 0 1px black,
                      inset 0 0 0 2px transparent"})
-     (cond (= (time/format-date-iso-8601 date1) d)
+     (cond (and (= (time/format-date-iso-8601 date1) d) (nil? date2))
+           (style/date1-highlight-style hash-color colors/gray800)
```

Comment:
Nitpicking: define logical styles `day-neutral`, `day-compare-1st` and `day-compare-2nd`?